.github/workflows/ms.devtestlab.labs.yml

```diff
@@ -19,14 +19,14 @@ on:
       - main
     paths:
       - '.github/actions/templates/**'
+      - '.github/workflows/template.module.yml'
       - '.github/workflows/ms.devtestlab.labs.yml'
       - 'modules/Microsoft.DevTestLab/labs/**'
       - 'utilities/pipelines/**'
       - '!utilities/pipelines/deploymentRemoval/**'
       - '!*/**/readme.md'
 
 env:
-  variablesPath: 'settings.yml'
   modulePath: 'modules/Microsoft.DevTestLab/labs'
   workflowPath: '.github/workflows/ms.devtestlab.labs.yml'
   AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
```

Comment:
Same here. We should be able to remove those too

Reply:
done